visitors.html

```diff
@@ -1,183 +1,171 @@
-          <a class="btn btn-danger" href="http://shop.gorkogo55.ru/search"
-            >ВСЕ МАГАЗИНЫ ТУТ</a
-          >
-        </p>
         <li>
-          <p class="text-justify">
-            Наш торговый центр находится по адресу Горького 55, Калининград. От
-            площади Победы пройти на остановку «Улица Черняховского» и сесть на
-            троллейбус №7 или любой из автобусов с номерами 1, 25, 30, 48 и 7.
-            Выйти на остановке «Детская Больница». Перейти на пешеходном
-            переходе улицу Горького и пройти 100 метров в обратную сторону.
-            Поздравляем, вы на месте!
-          </p>
+          <span>
+            <i class="far fa-clock text-secondary"></i>
+          </span>
+          с 09:00 до 19:00
+          <span id="text-open" class="text-success d-none">
+            (Сейчас открыто)
+          </span>
         </li>
       </ul>
     </div>
-    <footer class="container pb-5">
+  </div>
 
-      <div>
-        <!-- Yandex.Metrika informer -->
-        <a
-          href="https://metrika.yandex.ru/stat/?id=53046907&amp;from=informer"
-          target="_blank"
-          rel="nofollow"
-        >
-          <img
-            src="https://informer.yandex.ru/informer/53046907/3_1_FFFFFFFF_FFFFFFFF_0_pageviews"
-            style="width:88px; height:31px; border:0;"
-            alt="Яндекс.Метрика"
-            title="Яндекс.Метрика: данные за сегодня (просмотры, визиты и уникальные посетители)"
-            class="ym-advanced-informer"
-            data-cid="53046907"
-            data-lang="ru"
-          />
-        </a>
-        <!-- /Yandex.Metrika informer -->
-      </div>
-    </footer>
+  <div class="container pb-3">
+    <ul class="list-unstyled">
+      <li>
+        <p>
+          Дорогие посетители! Просим Вас ознакомится с актуальными акциями и
+          скидками:
+        </p>
+      </li>
+      <p class="text-center">
+        <a class="btn btn-danger" href="http://shop.gorkogo55.ru/search">ВСЕ МАГАЗИНЫ ТУТ</a>
+      </p>
+      <li>
+        <p class="text-justify">
+          Наш торговый центр находится по адресу Горького 55, Калининград. От
+          площади Победы пройти на остановку «Улица Черняховского» и сесть на
+          троллейбус №7 или любой из автобусов с номерами 1, 25, 30, 48 и 7.
+          Выйти на остановке «Детская Больница». Перейти на пешеходном
+          переходе улицу Горького и пройти 100 метров в обратную сторону.
+          Поздравляем, вы на месте!
+        </p>
+      </li>
+    </ul>
+  </div>
 
-    <script type="text/javascript">
-      // https://stackoverflow.com/a/800010/1775459
-      document.addEventListener("DOMContentLoaded", function() {
-        var element = document.getElementById("text-open");
-        if (element) {
-          var kaliningradHours = new Date().getUTCHours() + 2;
-          var isMallOpened = kaliningradHours >= 9 && kaliningradHours < 19;
+  <div class="container pt-3 pb-3 text-center">
```

Comment:
Here like and other similar places you used pt and pb classes with the same size. Instead of this you can use only one py-3 which has the same functionality :)